client/src/pages/Login.js

```diff
@@ -40,12 +45,30 @@ class LoginPage extends Component {
 
   handleSubmit = event => {
     event.preventDefault();
-    const isValid = this.validate();
-    if (isValid) {
-      console.log(this.state);
-      this.setState(initalState);
+    this.validate();
```

Comment:
In this case 'this.validate()' will just return true or false. You need the result to decide whether to send the data to the server or not. That is, you only send the data if it returns true.